models/user.py

```diff
@@ -0,0 +1,15 @@
+from flask_login import UserMixin
+from .role import Role
+
+class User(UserMixin):
+    def __init__(self, name, id, active=True, role=Role.READER):
+        self.name = name
```

Comment:
I can't see anywhere you use this name field, and when this is called you just pass through a 'name' string - can we just lose that?